tests/NRedisStack.Tests/Examples/ExampleTests.cs

```diff
@@ -278,7 +278,7 @@ public async Task TransactionExample()
         Assert.Equal("[1200]", totalAmtOfShachar.Result.ToString());
     }
 
-    [Fact]
+    [SkipIfRedis(Is.Cluster)]
```

Comment:
SkipifRedis(Is.OSSCluster)?

WDYT? There are different cluster types (enterprise)